File: hotel_search/v2/by_hotel/ios/HotelSearch.swift
import Amadeus

var amadeus: Amadeus = Amadeus(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

// Get list of offers for a specific hotel
amadeus.shopping.hotelOfferByHotel.get(params: ["hotelId": "BGMILBGB",
                                                "adults": "2",
                                                "roomQuantity": "1",
                                                "paymentPolicy": "NONE"],
                                       onCompletion: {
                                           response, error in
                                           if error == nil {
                                               print(response!.data)
                                           }
                                        })

File: hotel_search/v2/get/hotel_offers_by_hotel/Node SDK/hotel_search.js
var Amadeus = require("amadeus");
var amadeus = new Amadeus({
  clientId: 'YOUR_API_KEY',
  clientSecret: 'YOUR_API_SECRET'
});

// Get list of offers for a specific hotel
amadeus.shopping.hotelOffersByHotel.get({
  hotelId: 'BGMILBGB'
}).then(function (response) {
  console.log(response);
}).catch(function (response) {
  console.error(response);
});